internal/command/image/show.go

```diff
@@ -91,11 +134,104 @@ func runShow(ctx context.Context) error {
 		},
 	}
 
-	return render.VerticalTable(io.Out, "Deployment Status", obj,
+	return render.VerticalTable(io.Out, "Image Details", obj,
+		"Registry",
+		"Repository",
+		"Tag",
+		"Version",
+		"Digest",
+	)
+}
+
+func showMachineImage(ctx context.Context, app *api.AppCompact) error {
+	var (
+		client = client.FromContext(ctx).API()
+		io     = iostreams.FromContext(ctx)
+	)
+
+	// if we have machine_id as an arg, we want to show the image for that machine only
+	if len(flag.Args(ctx)) > 0 {
+		flaps, err := flaps.New(ctx, app)
+		if err != nil {
+			return err
+		}
+
+		raw, err := flaps.Get(ctx, flag.FirstArg(ctx))
+		if err != nil {
+			return fmt.Errorf("failed to get machine: %w", err)
+		}
+
+		var machine = new(api.V1Machine)
+
+		if err := json.Unmarshal(raw, machine); err != nil {
+			return fmt.Errorf("failed to unmarshal machine: %w", err)
+		}
+
+		fmt.Println(machine.ID)
```

Comment:
I don't know that this provides any value to the user since they provided the machine ID as part of the command. Unless you feel strongly, I'd prefer to remove it.

Reply:
Yep I'll remove that